src/test/java/uk/gov/pay/connector/it/resources/worldpay/WorldpayCardResourceIT.java

```diff
@@ -304,4 +309,45 @@ public void shouldReturnStatus500_AWorldpayPaResParseError() {
 
         assertFrontendChargeStatusIs(chargeId, AUTHORISATION_ERROR.getValue());
     }
+
+    @Test
+    public void shouldNotIncludeMachineCookiesMoreThanOne() {
```

Comment:
Don't think the test asserts that it is set exactly once. Instead, it is testing the machine cookie is being set.

```suggestion
    public void shouldIncludeMachineCookieInTheRequestHeaderInThe3dsAuthorisationRequestToWorldpay() {
```

Reply:
👍